gfw_pixetl/utils/utils.py

```diff
@@ -220,3 +221,22 @@ def union(
             geom = MultiPolygon([geom])
 
     return geom
+
+
+def enumerate_bands(num_bands: int) -> List[str]:
+    """Return a variable name for each of num_bands."""
+
+    def count_with_letters():
+        """Generate an infinite sequence of strings of uppercase letters
+        corresponding to numbers in base 26.
+
+        Taken from https://stackoverflow.com/a/29351603.
+        """
+        for size in itertools.count(1):
+            for letters in itertools.product(string.ascii_uppercase, repeat=size):
+                yield "".join(letters)
+
+    band_names: List[str] = list()
+    for s in itertools.islice(count_with_letters(), num_bands):
```

Comment:
This seems dangerous. Don't we need to be careful with the value of `num_bands` to keep from going into an infinite loop?
from the docs:
```
If stop is None, then iteration continues until the iterator is exhausted, if at all; otherwise, it stops at the specified position.
```

Reply:
You're right, the type hint is only a hint! I'll protect us with an isinstance and a ValueError exception.